.copier-answers.yml

```diff
@@ -14,7 +14,7 @@ odoo_version: 16.0
 org_name: Odoo Community Association (OCA)
 org_slug: OCA
 rebel_module_groups: []
-repo_description: 'TODO: add repo description.'
+repo_description: Addons concerning Odoo Italian Localization.
 repo_name: l10n-italy
```

Comment:
Grazie della PR!
Possiamo cambiare anche il nome del repository come in https://github.com/OCA/l10n-italy/pull/3005?
La domanda di `copier` dice che dovrebbe essere _human-readable_:
> Tell me the project name. It's supposed to be human-readable. So, server-tools project could be named like "Tools for server environment(s)"

Reply:
cambiato

Reply:
@SirTakobi ho accettato il tuo suggerimento: puoi verificare se è ok? Grazie.

Reply:
> @SirTakobi ho accettato il tuo suggerimento: puoi verificare se è ok? Grazie.

sì è ok